te_markdown/define-password.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: "Tutorial: Define Passwords"
+permalink: /te_markdown/define-passwords/
+---
+
+Estimated reading time: 5 minutes
+
+## Motivation
+
+Often, applications under test require user credentials like username and password for user input. In order not to store user information directly in the test case descriptions in plain text, viewable for all test participants, the Test-Editor offers the possibility to make these data available to the test cases with following approaches. 
```

Comment:
"In order not to store user information" -> "In order to avoid storing user information"